libcodechecker/server/api/report_server.py

```diff
@@ -2239,9 +2239,18 @@ def massStoreRun(self, name, tag, version, b64zip, force,
                 source_root = os.path.join(zip_dir, 'root')
                 report_dir = os.path.join(zip_dir, 'reports')
                 metadata_file = os.path.join(report_dir, 'metadata.json')
+                skip_file = os.path.join(report_dir, 'skip_file')
                 content_hash_file = os.path.join(zip_dir,
                                                  'content_hashes.json')
 
+                if os.path.exists(skip_file):
+                    LOG.debug("Pocessing skip file %s", skip_file)
+                    with open(skip_file) as sf:
```

Comment:
If it throws an exception, `skiphandler` variable will be undefined. You should initialize this variable before the condition checking.